src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final String[] colours = new String[]{
```

Comment:
```suggestion
    public static final String[] COLOURS = new String[]{
```
constants should have UPPER_CASE_NAMES